student.rb

```diff
@@ -0,0 +1,10 @@
+# Student is a class representing a student, which inherits from Person.
+class Student < Person
+  def initialize(age, _classroom, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission: parent_permission)
+  end
+
```

Comment:
- [ ] Kindly make sure that When setting the classroom for a student it also adds it to the classrooms' students, you can create a new method for that 
e.g
```ruby
def classroom=(classroom)
...
...
end
```